ramsey_cop.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(/^exe\//) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "rubocop", ">= 0.54", "< 0.59"
+  spec.add_dependency "rubocop", ">= 0.54", "< 0.60"
```

Comment:
@jwsloan @mellowfish 

John, I know you saw this and commented on a previous Dependabot PR that we should probably just let Dependabot do its thing.

I just want to call out though that we should probably keep an eye on this. I am wondering if it is doing this strangeness because I had `"~> 0.54.0"` in the gemspec before Rubocop took over.

I don't want it to get stuck at `< 0.60` edit: but I do want to see what Dependabot does.

Reply:
I have mostly slapped down dependabot changes away from `~>` syntax. Don't let it do things you don't agree with.